lib/kibana/kibana/connector.py

```diff
@@ -89,7 +89,7 @@ def url(self, uri):
         # If a space is defined update the URL accordingly
         uri = uri.lstrip('/')
         if self.space:
-            uri = "s/{}/{}".format(self.space, uri)
+            uri = "s/{}/{}".format(self.space.lower(), uri)
```

Comment:
it might be better to use [urljoin](https://docs.python.org/3/library/urllib.parse.html#urllib.parse.urljoin) here instead of manual string formatting, but we would need to check that its behaviour with left-side `/` is as expected

Reply:
it's definitely a nitpick